conode/Makefile

```diff
@@ -5,10 +5,13 @@ TAG=$(shell git describe --tags)
 OUTPUT_DIR=conode-$(TAG)
 # dev is the master branch, which will get automatic date tagging for
 # distribution files.
-ifeq ($(TAG),dev)
-	TAG := $(TAG)-$(shell date +%y%m%d)
+STRIPPED_TAG = $(shell echo $(TAG) | cut -d '-' -f 1)
+VERSION := $(STRIPPED_TAG)
+ifeq ($(findstring v2,$(STRIPPED_TAG)),v2)
```

Comment:
this thing in fact did not work out, as we'll need to add a `v2` tag, but that's interfering with the `v2` branch :( Any idea how to know if we're sitting somewhere on the `v2` branch?

Reply:
I think this part might be a result of a merge. I can change it back to what it was previously. Would that be better? But doing that, I will have to hard-code the docker tag for `docker_dev`.